tsdoc/src/transforms/DocNodeTransforms.ts

```diff
@@ -0,0 +1,43 @@
+import { TrimSpacesTransform } from './TrimSpacesTransform';
+import { DocParagraph, DocNode } from '../nodes';
+
+/**
+ * Helper functions that transform DocNode trees.
+ */
+export class DocNodeTransforms {
+  /**
+   * The SpaceTrimmer collapses extra spacing characters from plain text nodes.
```

Comment:
>SpaceTrimmer [](start = 9, length = 12)

*`trimSpacesInParagraphNodes`

Reply:
I also updated the doc comment.

---
In reply to: [218215685](https://github.com/Microsoft/tsdoc/pull/61#discussion_r218215685) [](ancestors = 218215685)